vendor/assets/stylesheets/multi-select.scss

```diff
@@ -100,27 +89,17 @@
   cursor: text;
 }
 .ms-header{
-    font-weight: bold;
-    color:#ffffff;
-    background-color: #aaa;
-    padding: 4px 10px;
-    -webkit-border-radius: 4px 4px 0 0;
-    -moz-border-radius: 4px 4px 0 0;
-    border-radius: 4px 4px 0 0;
+  padding: 4px 10px;
+  border: 1px solid #d1d1d1;
+  background-clip: padding-box;
+  @include linear-gradient(#f9f9f9,#ededed);
```

Comment:
This causes the +/- in the multiselect headers to be almost hidden, you only notice them if you happend to hover over them.